pkg/datapath/loader/loader.go

```diff
@@ -274,6 +274,24 @@ func (l *Loader) reloadHostDatapath(ctx context.Context, ep datapath.Endpoint, o
 	return nil
 }
 
+func replaceDatapathAndHandleError(ctx context.Context, ep datapath.Endpoint, objPath, progSec, progDirection string, finalizeChan chan func(), errChan chan error) {
```

Comment:
I have a mixed feelings here. I'd wrap it into go rountine here instead of just pushing results to chans.

Reply:
Wrapped it in a goroutine here